### ISSUE TITLE
%zu: use SIZE_FMT_SPECIFIER instead

### ISSUE THREAD

**Issue by vtorri** [pkgconf/pkgconf#273]
the purpose of `SIZE_FMT_SPECIFIER` is to give a crossplatform specifier of %zu, so what about using it ?

note also that in client.c,

```
libpkgconf/queue.c:             PKGCONF_TRACE(client, "slot %zu: dep %s matched to %p<%s> hits %lu", i, dep->package, dep->match, dep->match->id, dep->match->hits);
```

last format specifier should be a %zu, not %lu

**Comment by vtorri** [comment]
i can provide a PR if you want

**Comment by kaniini** [comment]
Fixed in 99bc93f
